src/cmds/scripts/pbs_snapshot

```diff
@@ -53,44 +53,48 @@ if [ ! -d  ${ptl_prefix_lib} ]; then
     fi
 fi
 
+conf="${PBS_CONF_FILE:-/etc/pbs.conf}"
+if [ -r "${conf}" ]; then
+    # we only need PBS_EXEC from pbs.conf
+    __PBS_EXEC=$( grep '^[[:space:]]*PBS_EXEC=' "$conf" | tail -1 | sed 's/^[[:space:]]*PBS_EXEC=\([^[:space:]]*\)[[:space:]]*/\1/' )
+fi
+echo $__PBS_EXEC
+unset conf
+
 if [ "x${ptl_prefix_lib}" != "x" ]; then
-        python_dir=$(/bin/ls -1 ${ptl_prefix_lib})
-        prefix=$(dirname ${ptl_prefix_lib})
+    python_dir=$(/bin/ls -1 ${ptl_prefix_lib})
+    prefix=$(dirname ${ptl_prefix_lib})
 
-        ptllibpath=${prefix}/lib/${python_dir}/site-packages
-        ptlbinpath=${prefix}/bin
+    ptllibpath=${prefix}/lib/${python_dir}/site-packages
+    ptlbinpath=${prefix}/bin
 else
-        conf="${PBS_CONF_FILE:-/etc/pbs.conf}"
-        if [ -r "${conf}" ]; then
-                # we only need PBS_EXEC from pbs.conf
-                __PBS_EXEC=$( grep '^[[:space:]]*PBS_EXEC=' "$conf" | tail -1 | sed 's/^[[:space:]]*PBS_EXEC=\([^[:space:]]*\)[[:space:]]*/\1/' )
-                if [ "X${__PBS_EXEC}" != "X" ]; then
-                        # Define PATH and PYTHONPATH for the users
-                        PTL_PREFIX=$(dirname ${__PBS_EXEC})/ptl
-                        python_dir=$(/bin/ls -1 ${PTL_PREFIX}/lib)/site-packages
-                        if [ -d "${PTL_PREFIX}/bin" ];then
-                            ptllibpath=${PTL_PREFIX}/lib/${python_dir}
-                        fi
-                        if [ -d "${PTL_PREFIX}/lib/${python_dir}" ];then
-                            ptlbinpath=${PTL_PREFIX}/bin
-                        fi
-                fi
-                unset __PBS_EXEC
-                unset PTL_PREFIX
-                unset conf
-                unset python_dir
+    if [ "X${__PBS_EXEC}" != "X" ]; then
+        # Define PATH and PYTHONPATH for the users
+        PTL_PREFIX=$(dirname ${__PBS_EXEC})/ptl
+        if [ -x "${PTL_PREFIX}/lib)/site-packages" ]; then
+            python_dir=$(/bin/ls -1 ${PTL_PREFIX}/lib)/site-packages
         fi
+        if [ -d "${PTL_PREFIX}/bin" ];then
+            ptllibpath=${PTL_PREFIX}/lib/${python_dir}
+            ptlbinpath=${PTL_PREFIX}/bin
+        fi
+       unset PTL_PREFIX
+       unset python_dir
+    fi
 fi
 
 export PYTHONPATH=${ptllibpath}:${PYTHONPATH}
 
 if [ -z $ptlbinpath ];then
-    echo "Error: PTL bin path not found"
-    exit 1
-fi
-
-if [ -x "${PBS_EXEC}/python/bin/python" ]; then
-	${PBS_EXEC}/python/bin/python ${ptlbinpath}/pbs_snapshot "${@}"
+    if [ -x "${__PBS_EXEC}/python/bin/python" ]; then
```

Comment:
Don't run pbs_snapshot.py directly, please use the change that I suggested instead of making any of the following changes